.travis.yml

```diff
@@ -1,14 +1,13 @@
 language: java
 sudo: false
 script: 
-  - git clone https://github.com/SupunArunoda/CPACEP.git
+  - cd ../CPACEP
   - mvn clean verify
-  - mvn test
   
-
-before_script:
+before_install:
+  - cd ..
```

Comment:
Hi Thomas,
I think cd.. command is needed because default travis directory is SupunArunoda/CPACEP. It becomes bit complex to keep both CPCEP and cpachecker in same directory. That's why I have done this.

Reply:
Why does it become complex?

Reply:
Hi Thomas,
If we do the ant build first and then maven verify second, travis will use cpachecker as the default folder and try to do the mvn verify there too. Then the build error occurs because it cannot find the root pom. This has happened to me several times. Thats why I created separate directory.

Reply:
Is it fine to do all the things in script: body ?

Reply:
No, we want separation of concerns.

Reply:
Hi Thomas,
Where we keep the output generated by cpachecker without creating a directory?

Reply:
Hi Thomas,
How abut this?

language: java
sudo: false
script:
  - mvn clean verify

install:
  - git clone https://github.com/sosy-lab/cpachecker.git
  - mkdir /tmp/original
  - cd cpachecker
  - ant build
  - scripts/cpa.sh -spec config/specification/default.spc -config config/default.properties doc/examples/example.c -outputpath /tmp/original -stats
  - cd..

Reply:
Almost ok :wink:
You had something like this before and I told you that the execution of CPAchecker doesn't belong to `install`.
I would still try to split the unit tests of CPACEP and the integration test with CPAchecker,
but if this is too complicated for now,
how about putting the execution of cpachecker into `before_script` ?
(or just leave it out, it's not really necessary, after all)